verified-answers.yaml

```
@@ -14,3 +16,1551 @@
       title: MongoDB Atlas Registration
     - url: https://www.mongodb.com/basics/mongodb-atlas-tutorial?tck=mongodb_ai_chatbot
       title: MongoDB Atlas Tutorial
+- answer: >-
+    To get started with MongoDB, you can try the 5-minute interactive tutorial
+    that connects you to a MongoDB Atlas cluster and walks you through
+    inserting, querying, and deleting data in MongoDB. You do not need to
+    install anything for this tutorial. Simply click the Launch button of the
+    in-browser Integrated Development Environment to start the tutorial.
+
+    After completing the tutorial, you can also deploy a free cluster without any installation overhead by following the instructions in the "Getting Started with Atlas" documentation.
+
+    Let me know if you have any other questions!
+  author_email: ian.fogelman@mongodb.com
+  questions:
+    - Get started with MongoDB
+  references:
+    - url: https://mongodb.com/docs/manual/tutorial/getting-started/
+      title: Getting Started with MongoDB
+    - url: https://learn.mongodb.com/learn/course/start-here-introduction-to-mongodb/welcome-to-the-introduction-to-mongodb-course/learn
+      title: Welcome to the Introduction to MongoDB Course
+    - url: https://mongodb.com/docs/develop-applications
+      title: Develop Applications
+    - url: https://mongodb.com/docs/manual/
+      title: MongoDB Manual
+- answer: >-
+    To install MongoDB on Windows, you can follow these steps:
+
+    1. Download the MongoDB Community Edition `.msi` installer from the MongoDB
+      Download Center. Make sure to select the appropriate version, platform, and
+      package type.
```

```suggestion
    1. Download the MongoDB Community Edition `.msi` installer from the MongoDB
       Download Center. Make sure to select the appropriate version, platform, and
       package type.
```

nitty - IMO bulleted lists should use three spaces for multi-line list items so that everything lines up. Noticed this throughout the verified answers so probably something in the importer.

Doesn't _really_ matter though since this is technically within the spec as-is and will render just fine.

---

I had to format these myself actually. They weren't right at all in the original. Anyway, between 2 and 3 spaces, not a strong preference. I used 2 because it lets me use tab to block indent